cmd/dashboard.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"github.com/urfave/cli"
+
+	"github.com/Scalingo/cli/appdetect"
+	"github.com/Scalingo/cli/apps"
+	"github.com/Scalingo/cli/cmd/autocomplete"
+)
+
+var (
+	dashboardCommand = cli.Command{
+		Name:     "dashboard",
+		Category: "App Management",
+		Flags:    []cli.Flag{appFlag},
+		Usage:    "Open app dashboard on default web browser",
+		Description: `Open app dashboard on default web browser:
+
+	$ scalingo --region osc-fr1 --app my-app dashboard`,
+		Action: func(c *cli.Context) {
+			if len(c.Args()) != 0 {
+				cli.ShowCommandHelp(c, "dashboard")
+				return
+			}
+
+			if c.GlobalString("region") == "" {
+				cli.ShowCommandHelp(c, "dashboard")
+			}
+
+			currentApp := appdetect.CurrentApp(c)
+			err := apps.Dashboard(currentApp, c.GlobalString("region"))
+			if err != nil {
+				errorQuit(err)
+			}
```

Comment:
I don't think this is the way to get the current region. You can have a look at the file `cmd/commands.go`, the method `addCommand`. We already set the region in `config.C.ScalingoRegion`. I'm not completely sure about it but I think it should be replaced with:

```suggestion
			currentApp := appdetect.CurrentApp(c)
			err := apps.Dashboard(currentApp, config.C.ScalingoRegion)
			if err != nil {
				errorQuit(err)
			}
```

Reply:
Thanks for the tip, I spent hours trying to figure this one out since region is set and I couldn't figure out how to get the value of it. Gonna fix it.